WcaOnRails/db/migrate/20170403170418_create_rounds.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRounds < ActiveRecord::Migration[5.0]
+  def change
+    create_table :rounds do |t|
+      t.integer :competition_event_id, null: false
+      t.string :format_id, null: false
+      t.integer :nth_round, null: false
```

Comment:
I see. I can tell you that I've already spent enough time thinking about this that that is not the case for me, but I'm not a valid person to ask anymore. For lack of a better plan, I'll reach out via slack and see what people say.